manifests/service.pp

```diff
@@ -0,0 +1,46 @@
+# Class: shibboleth_idp::service
+#
+# Manages the Jetty process
+#
+# Sample Usage:
+#
+#    sometype { 'foo':
+#      notify => Class['shibboleth_idp::service'],
+#    }
+#
+#
+class shibboleth_idp::service (
+  $service_name    = $::shibboleth_idp::params::service_name,
+  $service_enable  = $::shibboleth_idp::params::service_enable,
+  $service_ensure  = $::shibboleth_idp::params::service_ensure,
+  $service_manage  = $::shibboleth_idp::params::service_manage,
+  $service_restart = $::shibboleth_idp::params::service_restart,
+) {
```

Comment:
I meant don't have any params at all in the service class, like in the install class:
https://github.com/MiamiOH/puppet-shibboleth_idp/blob/1c75e07ff772a8165588ef3acef307014dc78eef/manifests/install.pp#L6

Just read the vars directly from the init class. Like this:
https://github.com/MiamiOH/puppet-shibboleth_idp/blob/master/manifests/install.pp#L24
The init class is the interface the end user should use. They should set any params there only.

If the service class is private and included by the init class, noone should set params there.